features/FEATURE_BLE/ble/BLE.h

```diff
@@ -400,6 +400,15 @@ class BLE
      */
     const SecurityManager& securityManager() const;
 
+    /**
+     * Translate error code into a printable string.
+     *
+     * @param[in] error Error code returned by BLE functions.
+     *
+     * @return A pointer to a const string describing the error.
+     */
+    static const char* errorToString(ble_error_t error);
```

Comment:
I'm confused if we should stick with BLE API conventions or start to use mbed OS convention. 
I suppose global alignment  with mbed-os code base must happen at once.